apps/sherbet_service/config/config.exs

```diff
@@ -1,7 +1,9 @@
 use Mix.Config
 
 config :sherbet_service,
-    ecto_repos: [Sherbet.Service.Repo]
+    ecto_repos: [Sherbet.Service.Repo],
+    mobile_key_length: 8,
+    email_key_length: 32
```

Comment:
This shouldn't be changed, as these configs will only be optional it should simply be left up to the user of this service to determine whether they should override this default behaviour or not.